frontend/src/views/WheresMyGene/components/HeatMap/utils.ts

```diff
@@ -326,10 +368,43 @@ export function getHeatmapHeight(cellTypes: CellTypeSummary[]): number {
   return HEAT_MAP_BASE_HEIGHT_PX + HEAT_MAP_BASE_CELL_PX * cellTypes.length;
 }
 
+/**
+ * Value format: `${id}~${tissue}~${name}`
+ * We need to encode cell type metadata here, so we can use it in onClick event
+ */
 export function getCellTypeNames(cellTypes: CellTypeSummary[]): string[] {
-  return cellTypes.map((cellType) => cellType.name);
+  return cellTypes.map(({ id, name }) => {
+    // TODO(thuang): Add multiple tissues support.
+    return `${id}~lung~${name}`;
```

Comment:
Hard coding `lung` right now, but in the next PR I'll add more multiple tissue support!